bcgov/HOUS-permit-portal
# Release/0.2.0

## Description

Fixes clearing of PIDs

## What type of PR is this? (check all applicable)

- [ x] 🐛 Bug Fix

## Related Tickets & Documents
https://hous-bssb.atlassian.net/browse/BPHH-1737

## Steps to QA
create a new application
add a PID into the PID field.  Hit the x, it should clear the PID, and will clear out all the selections below as well if it was a valid PID.
